api/envoy/extensions/filters/http/sxg/v3alpha/sxg.proto

```diff
@@ -0,0 +1,56 @@
+syntax = "proto3";
+
+package envoy.extensions.filters.http.sxg.v3alpha;
+
+import "envoy/extensions/transport_sockets/tls/v3/secret.proto";
+
+import "google/protobuf/duration.proto";
+
+import "udpa/annotations/status.proto";
+
+option java_package = "io.envoyproxy.envoy.extensions.filters.http.sxg.v3alpha";
+option java_outer_classname = "SxgProto";
+option java_multiple_files = true;
+option (udpa.annotations.file_status).work_in_progress = true;
+option (udpa.annotations.file_status).package_version_status = ACTIVE;
+
+// [#protodoc-title: Signed HTTP Exchange Filter]
+// SXG :ref:`configuration overview <config_http_filters_sxg>`.
+// [#extension: envoy.filters.http.sxg]
+
+// [#next-free-field: 10]
+message SXG {
+  // SDS key for x509 certificate data
+  transport_sockets.tls.v3.SdsSecretConfig certificate = 1;
+
+  // SDS key for private key data.
+  transport_sockets.tls.v3.SdsSecretConfig private_key = 2;
```

Comment:
can you comment a bit more on how these two will be used? Maybe with a link to the SXG docs explaining how it's used, etc.